docs/src/transformations.md

```diff
@@ -14,41 +14,300 @@ transform!
 ```
 
 # Example
-Here we show how can we reduce the complexity of the tensor network by applying a tranformation to it:
+Here we show how can we reduce the complexity of the tensor network by applying a tranformation to it. We take as an example the Sycamore circuit from the [Google's quantum supremacy paper](https://www.nature.com/articles/s41586-019-1666-5)
+
 ```@setup plot
+using Makie
+Makie.inline!(true)
+
 using CairoMakie
-CairoMakie.activate!(type = "svg")
+using Tenet
+using NetworkLayout
 
 using Pkg
 Pkg.add("QuacIO")
+
+function smooth_annotation!(f; color=Makie.RGBAf(110 // 256, 170 // 256, 250 // 256, 60 // 256), xlims=[-2, 2], ylims=[-2, 2], offset_x=0.0, offset_y=0.0, radius_x=1.0, radius_y=1.0, num_waves=5, fluctuation_amplitude=0.1, phase_shift=0.0)
+    ax = Axis(f)
+    hidedecorations!(ax)
+    hidespines!(ax)
+
+    # Define limits of the plot
+    xlims!(ax, xlims...)
+    ylims!(ax, ylims...)
+
+    # Create a perturbed filled shape
+    theta = LinRange(0, 2π, 100)
+
+    fluctuations = fluctuation_amplitude .* sin.(num_waves .* theta .+ phase_shift)
+
+    # Apply the fluctuations and radius scaling
+    perturbed_radius_x = radius_x .+ fluctuations
+    perturbed_radius_y = radius_y .+ fluctuations
+
+    circle_points = [Point2f((perturbed_radius_x[i]) * cos(theta[i]) + offset_x,
+                              (perturbed_radius_y[i]) * sin(theta[i]) + offset_y) for i in 1:length(theta)]
+
+    poly!(ax, circle_points, color=color, closed=true)
+end
+
+bg_blue = Makie.RGBAf(110 // 256, 170 // 256, 250 // 256, 50 // 256) #hide
+orange = Makie.RGBf(240 // 256, 180 // 256, 100 // 256) #hide
+red = Makie.RGBf(240 // 256, 90 // 256, 70 // 256) #hide
 ```
-```@example transformation
+```@example plot
 using QuacIO
-using CairoMakie
-using Tenet
+set_theme!(resolution=(800,400)) # hide
 
 sites = [5, 6, 14, 15, 16, 17, 24, 25, 26, 27, 28, 32, 33, 34, 35, 36, 37, 38, 39, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 61, 62, 63, 64, 65, 66, 67, 72, 73, 74, 75, 76, 83, 84, 85, 94]
 circuit = QuacIO.parse(joinpath(@__DIR__, "sycamore_53_10_0.qasm"), format=QuacIO.Qflex(), sites=sites)
 tn = TensorNetwork(circuit)
 transformed_tn = transform(tn, Tenet.RankSimplification)
```

Comment:
Would you mind adding some more transformations here? I know that quimb's `full_simplify` function uses `AntiDiagonalGauging`, `DiagonalReduction`, `ColumnReduction` and `RankSimplification` in that order by default.

Reply:
Sure!

Reply:
I have done that but it appears that for this specific case we obtain the same network that we would obtain with only `RankSimplification`. Nevertheless, we would expect some advantage for some larger circuits when we apply all the transformations.